### PR TITLE
cmd: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/cmd.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# cmd
+
+> The Windows command interpreter.
+
+- Start a new instance of the command interpreter:
+
+`cmd`
+
+- Run the specified command and terminate:
```
Comment (sbrl) [inline pages/windows/cmd.md]: Perhaps ...`and then exit` would read better?

My thinking is that the example below could be altered to says something like `and then enter an interactive shell` or something.

Comment (pxgamer) [reply pages/windows/cmd.md]: Sorry, been away this weekend. :+1: I think that definitely is more clear. :slight_smile:

**Comment by pxgamer** [comment]
@sbrl @agnivade is this ok to be merged?

**Comment by sbrl** [comment]
Yep! I thought that @agnivade was going to merge it, but I've just done so anyway :-)

**Pull request merged**